content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/closest_location.rst

```diff
@@ -0,0 +1,101 @@
+========================
+Closest location removal
+========================
+
+For the *Closest Location* removal strategy, products are picked based on the alphanumeric order of
+storage location titles.
+
+This strategy requires businesses to store products that are ordered most often in locations closest
+to the output area. Then, these locations are named in alphanumeric order, with the closest
```

Comment:
This strategy can be applied to all products, regardless of how often they're ordered. While it would make sense to store the most commonly ordered products closer, that's outside the scope of the strategy, so I don't think it's worth mentioning.

Reply:
deleted the paragraph!